e-keithel-backend/db/migrate/20220917085758_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :password_digest
+
+      t.timestamps
```

Comment:
We should consider using other attributes in the users table like 
- email and 
- role for admin, buyer, and seller too!